goony.go

```diff
@@ -77,3 +82,20 @@ func outputFile(filename string) files.File {
 
 	return file
 }
+
+func startContentStream(options config.Options, file files.File, linesChannel chan string) {
+	if options.RunEndless == true {
+		stop := make(chan bool, 1)
+		catchCtrlC(stop)
+		go files.StreamContentForever(file, linesChannel, stop)
+	} else {
+		go files.StreamContent(file, linesChannel)
```

Comment:
I wonder if it would make sense to align `StreamContent` to `StreamContentForever`?
The CTRL-C functionality wouldn't hurt `StreamContent`, right?
Maybe you can even go as far as printing some useful stats to stderr after CTRL-C?